features/slack/assets/js/slack.js

```diff
@@ -0,0 +1,165 @@
+$('#slackchannels').on('click', 'span.close', function () {
+    $(this).fadeOut(100);
+    var row = $(this).parents('tr');
+    var newRow = "<tr><td colspan=\"2\"><div id=\"slack_placeholder\"/></tr></tr>";
+
+    newRow = $(newRow).insertAfter(row);
+    var placeholder = newRow.find("#slack_placeholder");
+
+    confirm_delete("Are you sure you want to delete this Slack channel?", placeholder, this, function() {
```

Comment:
might want to change the wording here so as to not imply it deletes the Slack channel, but just removes the channel log from the morgue entry

Reply:
I changed it to `Are you sure you want to remove slack channel logs from the morgue entry?`.